db/migrate/20170913131914_create_active_admin_comments.rb

```diff
@@ -0,0 +1,17 @@
+class CreateActiveAdminComments < ActiveRecord::Migration::Current
```

Comment:
Feel free to remove this class since we aren't using activeadmin comments. Remember to drop your db and rerun the migrations after this